--- arping-2.14/configure.ac.orig	2014-06-29 12:05:51.000000000 +0200
+++ arping-2.14/configure.ac	2014-07-04 20:21:49.765587884 +0200
@@ -17,8 +17,8 @@
 
 # Checks for libraries.
 AC_CHECK_LIB([m], [sqrt])
-AC_CHECK_LIB([socket], [socket])
-AC_CHECK_LIB([nsl], [gethostbyname])
+AC_SEARCH_LIBS([socket], [socket])
+AC_SEARCH_LIBS([gethostbyname], [nsl])
 AC_CHECK_LIB([cap], [cap_init])
 AC_CHECK_LIB([rt], [clock_gettime])
 AC_CHECK_LIB([net], [libnet_name_resolve],
